source/hp/fe_values.cc

```diff
@@ -135,7 +135,8 @@ namespace hp
   template <int dim, int q_dim, class FEValuesType>
   FEValuesBase<dim, q_dim, FEValuesType>::FEValuesBase(
     const FEValuesBase<dim, q_dim, FEValuesType> &other)
-    : fe_collection(other.fe_collection)
+    : Subscriptor(other)
```

Comment:
I have not used `Subscriptor`, yet. But why is here not passed in `*this`? To me it seems that you are registering the wrong class, do you?

Reply:
We should just call `Subscriptor()` here because we don't subscribe to the other object, we just store pointers to its data members (which is already set up correctly).

Reply:
It doesn't matter which constructor you call:
```
  /**
   * Constructor setting the counter to zero.
   */
  Subscriptor();

  /**
   * Copy-constructor.
   *
   * The counter of the copy is zero, since references point to the original
   * object.
   */
  Subscriptor(const Subscriptor &);
```